### PR TITLE
Include livestream overlay

### PR DESCRIPTION
**Pull request by WillB97**
Adds srobo/livestream-overlay to the install, update and run scripts

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
-for repo in dummy-comp sr2014-comp sr2015-comp sr2016-comp ranker srcomp srcomp-http srcomp-scorer srcomp-screens srcomp-stream srcomp-cli srcomp-kiosk; do
+#!/bin/bash
```
Comment (PeterJCLaw) [inline update.sh:1]: Does this strictly need `bash` or would `#!/bin/sh` suffice?

Comment (WillB97) [reply update.sh:1]: `pushd`/`popd` are specifically bash builtins, so `#!/bin/bash` is required.

File: run.py [hunk 2]
```diff
@@ -65,5 +67,16 @@ def run_streams():
...
+
+if args.overlay:
+    overlay_dir = os.path.realpath(os.path.join(mydir, 'livestream-overlay'))
+    config['/livestream-overlay'] = {
+        'tools.staticdir.on': True,
+        'tools.staticdir.dir': overlay_dir,
+        'tools.caching.on': False,
+        'tools.staticdir.index': 'stream.html',
+    }
+    print('Livestream overlay is available at: http://localhost:5112/livestream-overlay/')
```
Comment (PeterJCLaw) [inline run.py:79]: Is this print necessary? The user is going to know that they've passed the `--overlay` option.
If we do need this, please could we ensure that the url can't get out of step with the actual config?

Comment (WillB97) [reply run.py]: In hindsight, no. I've moved this information to the readme, alongside the other path information in 
6096ee9.

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**